interface/themes/labdata.css

```diff
@@ -0,0 +1,92 @@
+.graph {
```

Comment:
This file should be located in /interface/themes/misc and shouldn't be placed here. It also should have the `.scss` extension